src/fides/api/util/tcf/tc_model.py

```diff
@@ -446,6 +455,7 @@ def convert_tcf_contents_to_tc_model(
         vendor_consents=vendor_consents if consented else [],
         vendor_legitimate_interests=vendor_legitimate_interests if consented else [],
         vendors_disclosed=_build_vendors_disclosed(tcf_contents),
+        ac_vendor_consents=build_ac_vendor_consents(tcf_contents, preference),
```

Comment:
The AC Vendors from the vendor_consents section of the TCF Experience are added to the intermediate TC model.